### PR TITLE
Scheduler

### PR DESCRIPTION
**Pull request by Sail338**
- add components to have volunteer -> current action
- add a component to assign shifts
- remove old config
-remove package.lock.json

### CODE REVIEW

File: src/MasterTable.jsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    componentDidMount() {
+        const cookie = new Cookies();
+        const request_data = {
+            "email": cookie.get("email"),
+            "token": cookie.get("auth")
+        }
+        this.setState({
+            loading: true
+        })
+        httpClient.post("/getvolunteers", request_data).then(response => {
```
Comment (hemangandhi) [inline src/MasterTable.jsx:33]: You seem to assume a lot about the backend. Is the api documented somewhere?

File: src/MasterTable.jsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            }
+        })
+    }
+    //handler for changing the stafe on updating a row
+    onGridRowsUpdated = ({ fromRow, toRow, updated }) => {
+        this.setState(state => {
+          const rows = state.data.slice();
+          for (let i = fromRow; i <= toRow; i++) {
+            
+            this.state.data[i] = { ...rows[i], ...updated };
```
Comment (hemangandhi) [inline src/MasterTable.jsx:51]: Beautiful es6

**Review by hemangandhi — APPROVED** [review]
Nice!

File: src/MasterTable.jsx (hunk 1 above)
Comment (Sail338) [reply src/MasterTable.jsx:33]: i'll add comments in a coming PR that document the params for the backend

**Pull request merged**